mei/develop/mei-verovio_compiled.odd

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?><TEI xmlns:rng="http://relaxng.org/ns/structure/1.0" xmlns:sch="http://purl.oclc.org/dsdl/schematron" xmlns="http://www.tei-c.org/ns/1.0" xml:base="file:/Users/laurent/libs/mei_lpugin/mei-verovio.xml"><?TEIVERSION Version 4.0.0?>
+<?xml version="1.0" encoding="utf-8"?><TEI xmlns:rng="http://relaxng.org/ns/structure/1.0" xmlns:sch="http://purl.oclc.org/dsdl/schematron" xmlns="http://www.tei-c.org/ns/1.0" xml:base="file:/Users/davidbauer/Development/libmei/mei-verovio.xml"><?TEIVERSION Version 4.0.0?>
```

Comment:
I don't think these changes should be included.

Reply:
This is generated, but I can take it out manually.

Reply:
Yeah; it will just be 'churn' if everyone who touches it changes it (It's not actually ideal that it's coded to Laurent's paths either...)